src/current/cockroachcloud/aws-privatelink.md

```diff
@@ -1,128 +1,292 @@
 ---
-title: Managing AWS PrivateLink for a Dedicated Cluster
+title: Managing AWS PrivateLink for a cluster
 summary: Instructions for establishing AWS PrivateLink network access for a cluster.
 toc: true
 docs_area: manage
 ---
 
-Amazon Web Services (AWS) PrivateLink support in {{ site.data.products.dedicated }} allows customers to establish SQL access to their clusters entirely through private AWS infrastructure, without exposure to the public internet, affording enhanced security and performance.
+Amazon Web Services (AWS) PrivateLink support allows customers to establish SQL access to their clusters entirely through private AWS infrastructure, without exposure to the public internet, affording enhanced security and performance. PrivateLink is supported for {{ site.data.products.dedicated }} clusters and multi-region {{ site.data.products.serverless }} clusters deployed on AWS.
 
-For broader context, refer to [Network Authorization for CockroachDB Cloud Clusters](network-authorization.html).
+For broader context, refer to [Network Authorization for {{ site.data.products.db }} Clusters](network-authorization.html).
 
-This page describes the steps to setting up an AWS PrivateLink connection for your {{ site.data.products.dedicated }} cluster from your AWS VPC Console.
+This page describes the steps to setting up an AWS PrivateLink connection for {{ site.data.products.dedicated }} and multi-region {{ site.data.products.serverless }} clusters from your AWS VPC Console.
 
-{{site.data.alerts.callout_success}}
+<div class="filters clearfix">
+  <button class="filter-button" data-scope="dedicated">{{ site.data.products.dedicated }}</button>
+  <button class="filter-button" data-scope="serverless">Multi-region {{ site.data.products.serverless }}</button>
+</div>
+
+<section class="filter-content" markdown="1" data-scope="dedicated">
 
+{{site.data.alerts.callout_success}}
 You must also configure the AWS PrivateLink connection for your {{ site.data.products.dedicated }} cluster using the Console UI, Cloud API or Terraform Provider, so that {{ site.data.products.db }} can accept the new connection. For help, refer to [Establish VPC Peering or AWS PrivateLink](connect-to-your-cluster.html#establish-gcp-vpc-peering-or-aws-privatelink).
 
 If you have multiple clusters, you will have to repeat these steps for each cluster that you want to connect to using AWS PrivateLink.
 {{site.data.alerts.end}}
 
-### Set up a cluster
+</section>
+
+<section class="filter-content" markdown="1" data-scope="serverless">
+
+{{site.data.alerts.callout_success}}
+For each region where you deploy {{ site.data.products.serverless }} clusters with private connections, you must also configure the AWS PrivateLink connection for {{ site.data.products.serverless }} using the Console UI, Cloud API or Terraform Provider, so that {{ site.data.products.db }} can accept the new connection.
```

Comment:
Note that this also applies to multi-region dedicated clusters. Any reason why this was only called out in the serverless tab?

Reply:
In the Dedicated tab, it says instead:

```
You must also configure the AWS PrivateLink connection for your {{ site.data.products.dedicated }} cluster using the Console UI, Cloud API or Terraform Provider, so that {{ site.data.products.db }} can accept the new connection. For help, refer to [Establish VPC Peering or AWS PrivateLink](connect-to-your-cluster.html#establish-gcp-vpc-peering-or-aws-privatelink).

If you have multiple clusters, you will have to repeat these steps for each cluster that you want to connect to using AWS PrivateLink.
```

Is that not accurate?

Reply:
Yeah I think we're good here. I think Matt is being specific because in Serverless it doesn't apply to single-region clusters, which makes sense.